src/renderer/state.ts

```diff
@@ -326,6 +327,25 @@ export class AppState {
       this.toggleBisectCommands,
     );
     ipcRendererManager.on(IpcEvents.BEFORE_QUIT, this.setIsQuitting);
+    ipcRendererManager.on(
+      IpcEvents.DOWNLOAD_ELECTRON_CALLBACK,
+      this.updateDownloadProgress,
+    );
+    ipcRendererManager.on(
+      IpcEvents.SET_FIDDLE_RUNNING_STATE,
+      this.setRunningState,
+    );
+    ipcRendererManager.on(
+      IpcEvents.PUSH_OUTPUT_TO_CONSOLE,
+      this.pushOutputWrapper.bind(this),
+    );
+    ipcRendererManager.on(
+      IpcEvents.INSTALLER_STATE_CHANGE_EVENT,
+      (_, version: string, state: InstallState) => {
+        this.changeRunnableState(version, state);
+      },
+    );
```

Comment:
Since this is an async operation (handled via IPC), this triggers the version state change event after the electron version is actually unzipped. By that time, the code execution moves forward and auto-bisect interprets the version to be still `installing` (unzipping) and hence fails.
Figuring out a way to wait until this event is triggered (state is changed to `installed`) and then only move with rest of code execution.

Reply:
A workaround could be to manually trigger the `changeRunnableState` as soon as the version is unzipped. Will go ahead with that now :)